doc/whats_new/v1.0.rst

```diff
@@ -694,6 +694,10 @@ Changelog
   the final estimator.
   :pr:`19790` by :user:`Christopher Flynn <crflynn>`.
 
+- |Enhancement| Added support for "passthrough" in :class:`FeatureUnion`.
```

Comment:
We will need to move this entry in the `v1.1.rst` now that we released the RC

Reply:
I moved it